src/gluonnlp/data/utils.py

```diff
@@ -352,7 +358,7 @@ def line_splitter(s):
     return s.splitlines()
 
 def whitespace_splitter(s):
```

Comment:
This can probably be removed (superseded by Splitter)? Only reason for introducing it was that `lambda s: s.split()` can not be pickled and does not work in a multiprocessing data pipeline

Reply:
I'll track this separately in #425